editoast/editoast_schemas/src/paced_train.rs

```diff
@@ -24,4 +51,120 @@ pub struct PacedTrain {
     pub train_schedule_base: TrainSchedule,
     #[schema(inline)]
     pub paced: Paced,
+    #[serde(default)]
+    #[schema(required)]
+    pub exceptions: Vec<PacedTrainException>,
+}
+
+/// Represents an exception for a paced train occurrence.
+/// Occurrences are normally generated from a base paced train.
+/// An exception occurs when an occurrence is added or modified
+/// compared to the base model. Each field corresponds to a "ChangeGroup" of attributes
+/// that may deviate from the paced train.
+/// - Created: A new occurrence manually added by the user, not originally present in the automatically generated occurrences based on the paced train.
+/// - Modified: An existing occurrence that has been changed
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct PacedTrainException {
+    /// Unique key for the exception within the paced train, required and generated by the frontend.
+    pub key: String,
+    #[serde(flatten, default)]
+    #[schema(inline)]
+    pub exception_type: PacedTrainExceptionType,
+    #[serde[default]]
+    pub disabled: bool,
+    pub train_name: Option<TrainNameChangeGroup>,
+    pub rolling_stock: Option<RollingStockChangeGroup>,
+    pub rolling_stock_category: Option<RollingStockCategoryChangeGroup>,
+    pub labels: Option<LabelsChangeGroup>,
+    pub speed_limit_tag: Option<SpeedLimitTagChangeGroup>,
+    pub start_time: Option<StartTimeChangeGroup>,
+    pub constraint_distribution: Option<ConstraintDistributionChangeGroup>,
+    pub initial_speed: Option<InitialSpeedChangeGroup>,
+    pub options: Option<OptionsChangeGroup>,
+    pub path_and_schedule: Option<PathAndScheduleChangeGroup>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(untagged)]
+pub enum PacedTrainExceptionType {
+    Modified {
+        occurrence_index: i32,
+    },
+    /// Created is a struct variant (not unit) to allow proper deserialization with #[serde(untagged)].
+    /// This avoids ambiguity with an empty object `{}`.
+    /// Must be declared after more specific variants, as order matters for untagged enums.
+    Created {},
+}
+
+impl Default for PacedTrainExceptionType {
+    fn default() -> Self {
+        PacedTrainExceptionType::Created {}
+    }
+}
+
+impl ToSchema<'_> for PacedTrainExceptionType {
+    fn schema() -> (&'static str, RefOr<Schema>) {
+        let modified_schema = ObjectBuilder::new()
+            .property(
+                "occurrence_index",
+                ObjectBuilder::new().schema_type(SchemaType::Integer),
+            )
+            .build();
+        ("PacedTrainExceptionType", modified_schema.into())
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, ToSchema)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
```

or add it to all other changegroups (except the one with an f64)